File: webApi/Models/Destino.cs
using System;
using System.Collections.Generic;

namespace webApi.Models;

public partial class Destino
{
    public string Codigoaeropuertodestino { get; set; }

    public string Ciudad { get; set; }

    public string Pais { get; set; }

    public string Puertaingreso { get; set; }

    public DateTime? Horallegada { get; set; }


}

File: AdminWebApp/src/app/Models/Destino.ts
export class Destino {
    codigoAeropuerto: string;
    ciudad: string;
    puertaIngreso: string;
    pais: string;
  
    constructor(codigoAeropuerto: string, ciudad: string, puertaIngreso: string, pais: string) {
      this.codigoAeropuerto = codigoAeropuerto;
      this.ciudad = ciudad;
      this.puertaIngreso = puertaIngreso;
      this.pais = pais;
    }
  }